draft-ietf-rtcweb-jsep.xml

```diff
@@ -1165,10 +1165,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                         bundle-only. This policy aims to minimize
                         candidate gathering and maximize multiplexing,
                         at the cost of less compatibility with legacy
-                        endpoints. When acting as answerer, the implementation
-                        will accept only the compatible m= sections in the first bundle
-                        group, or, if the offer did not bundle any m=
-                        sections, only the first m= section.</t>
+                        endpoints. When acting as answerer, if there
+                        if no bundle group in the offer, the
+                        implementation will reject all but the first
+                        m= section and any other m= sections in the same
```

Comment:
This mention of a bundle group here seems confusing (since earlier in the sentence we have "if there is no bundle group in the offer"). What you had earlier was easier to follow in my opinion: "If there's a bundle group, do this, if there's no bundle group, do this".